Gobabygo/Gobabygo.ino

```diff
@@ -0,0 +1,74 @@
+const int joyStickXPin (0);
+// const int joyStickYPin (1);
+const int ledPin (6);
```

Comment:
instead of ledPin consider starting to name your variable to be in terms of motor (eg leftMotor and rightMotor.